dev/com.ibm.ws.jaxws.clientcontainer_fat/fat/src/com/ibm/ws/jaxws/clientcontainer/fat/FATSuite.java

```diff
@@ -25,4 +28,8 @@
                 WsBndServiceRefOverrideTest.class
 })
 public class FATSuite {
+
+    @ClassRule
+    public static RepeatTests r = RepeatTests.with(FeatureReplacementAction.NO_REPLACEMENT()).andWith(FeatureReplacementAction.EE9_FEATURES());
```

Comment:
Why isn't 2.3 part of this?  The auto feature mentioned earlier should make it work with 2.3 as well.

Reply:
Thomas explained that it uses the convenience feature so marking it resolved.